packages/expo-updates/android/build.gradle

```
@@ -6,7 +6,32 @@ apply plugin: 'maven-publish'
 group = 'host.exp.exponent'
 version = '0.18.5'
 
-def ex_updates_native_debug = System.getenv("EX_UPDATES_NATIVE_DEBUG") == "1" ? "true" : "false"
+// Utility method to derive boolean values from the environment or from Java properties,
+// and return them as strings to be used in BuildConfig fields
+def get_bool_string_value_from_prop_or_env(String name, Boolean defaultValue) {
+  def env_value = System.getenv(name)
+  def prop_value = findProperty(name)
+  def value = defaultValue.toString()
+  // If present, property value in gradle.properties overrides default
+  if (prop_value != null) {
+    value = boolish(prop_value).toString()
+    System.out.println("expo-updates: Value of ${name} was overridden by property, new value = ${value}")
```

can just use `println` (grepping in gradle files seems to indicate that this is preferred)

```suggestion
    println "expo-updates: Value of ${name} was overridden by property, new value = ${value}"
```